tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature

```diff
@@ -172,7 +172,7 @@ Feature: Create public link shares
       | uid_owner   | Alice          |
       | permissions | read           |
       | path        | /simple-folder |
-      | name        | Link           |
+      | name        | Quicklink      |
```

Comment:
might not need it: https://github.com/owncloud/web/issues/8929#issuecomment-1525030185